mesonbuild/backend/ninjabackend.py

```diff
@@ -14,6 +14,7 @@
 import json
 import os
 import pickle
+import plistlib
```

Comment:
This import is only needed on a single platform, and only some of the time. Maybe it would be a good idea to delay importing it until the function that uses it.

Reply:
This can be used on any platform (specifically I also want a reasonable build system for GNUstep on Linux). But yeah, I'll move the import into the function.